library/core/src/cell.rs

```diff
@@ -2364,6 +2370,9 @@ impl<T: CoerceUnsized<U>, U> CoerceUnsized<SyncUnsafeCell<U>> for SyncUnsafeCell
 //#[unstable(feature = "sync_unsafe_cell", issue = "95439")]
 impl<T: DispatchFromDyn<U>, U> DispatchFromDyn<SyncUnsafeCell<U>> for SyncUnsafeCell<T> {}
 
+#[unstable(feature = "pointer_like_trait", issue = "none")]
+impl<T: PointerLike> PointerLike for SyncUnsafeCell<T> {}
```

Comment:
Why not put these with the rest of the impls in `core::marker`?

Reply:
I did this because it is consistent with the existing code organization: impls of other marker traits for “primitive” types are in `core::marker` and defined using `marker_impls!`, and impls for library-defined types are next to those types. I’m not aware of a reason why `PointerLike` should be organized differently than other marker trait impls.

In fact, reviewing the existing usage of `marker_impls!`, I should probably put the impl for `NonNull` inside `core::ptr::non_null` instead of in the `marker_impls!` to be even more consistent.

Reply:
Sure, would you like to change that too?

Reply:
Done.